hatch-dependency-coversion/tests/conftest.py

```diff
@@ -0,0 +1,143 @@
+# SPDX-FileCopyrightText: 2024-present Opentrons Engineering <engineering@opentrons.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+from pathlib import Path
+import pytest
+
+
+@pytest.fixture
+def static_requirements() -> list[str]:
+    return [
+        "dependency1",
+        "dependency2==10.10.2",
+        "dependency3>=0.2.81; os_name == 'Windows'",
+    ]
+
+
+@pytest.fixture
+def unconfigured_project(tmp_path: Path, static_requirements: list[str]) -> Path:
+    (tmp_path / "pyproject.toml").write_text(
+        f"""
+[build-system]
+requires = ["hatchling", "hatch-dependency-coversion"]
+build-backend = "hatchling.build"
+[project]
+name = "unconfigured-project"
+version = "0.1.0"
+dependencies = [{','.join([f'"{requirement}"' for requirement in static_requirements])}]
+[tool.hatch.metadata.hooks.dependency-coversion]
+"""
+    )
```

Comment:
`textwrap.dedent()` can be helpful for things like this.

```suggestion
    (tmp_path / "pyproject.toml").write_text(textwrap.dedent(
        f"""
        [build-system]
        requires = ["hatchling", "hatch-dependency-coversion"]
        build-backend = "hatchling.build"
        [project]
        name = "unconfigured-project"
        version = "0.1.0"
        dependencies = [{','.join([f'"{requirement}"' for requirement in static_requirements])}]
        [tool.hatch.metadata.hooks.dependency-coversion]
        """
    ))
```